script/import_db.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+set -o errexit
+cd `dirname $0`/..
+
+DATABASE=READWRITE
+DIRECTORY=./dumps
+
+source ./admin/functions.sh
+
+if ! script/database_exists $DATABASE; then
+    count=`ls -l $DIRECTORY/mbdump*.tar.bz2 2>/dev/null | wc -l`
```

Comment:
There are a few other ways of doing this without running 2 extra commands, you could try one of the options here: https://stackoverflow.com/questions/2937407/test-whether-a-glob-has-any-matches-in-bash

If you stay with the current solution, you don't need the `-l` flag to ls (ls will automatically print one item per line if its output is a pipe). Also, you use quotes here, but `$()` in the command below to get the value of `LATEST`. These are equivalent and you should be consistent.